src/components/TodoPriority.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+    <select 
+        class="rounded-full text-white text-lg font-secondary
+            font-semibold text-center
+            w-32 appearance-none border-0 px-0 bg-none"
+        :class="selectedColor"
+        @change="updateColor"
+        v-model="selectedOption"        
+    >
```

Comment:
You can use v-for for the option element and use an object for the priority values and texts